ci/actions/install-autodiff-unix.sh

```diff
@@ -0,0 +1,10 @@
+export PATH=$HOME/miniconda/bin/:$PATH
+source activate autodiff
```

Comment:
you can use another approach here
you can do something like
```
source $HOME/miniconda/bin/activate
```
and after that you can use just ``conda activate autodiff``

This way is better because it will load the conda functions as well instead of just appending to the path

Reply:
The same is valid for your windows script

Reply:
@volpatto Have a look at this please.

Reply:
Yeah, actually I did some modifications in `autodiff`'s CI files. I would like to suggest to update this branch with `master`. Everything related with `CI`, when solving the conflicts, should be selected from `master`. Now we are using a built-in `miniconda` from GH Actions. Let's see what happens, I'm available to help in this regard.

Reply:
FYI: I merged this branch with master (#100), but compilation problems are still there. @tadeu helped me with that and found a bug with the compiler (already reported in https://developercommunity.visualstudio.com/content/problem/902908/compilation-problem-when-using-stdapply-with-eigen.html).

Reply:
@tadeu Knows all about MSVC! :)

Is there a way to change to a newer version, or an earlier version? Best would be to test here with the same version of MSVC used in autodiff-feedstock.

Reply:
I think that the bug is also present in the latest version, so we are out of luck with updating (until they release a fixed version), and we'll have to try to find a workaround.

The problem is with `std::apply`, it is not able to automatically convert an `Eigen::Matrix` to an `Eigen::Array`, even though when calling the function without `std::apply`, it works fine, and the parameter conversion is automatic.

Reply:
Thanks @tadeu - this should help. Perhaps I can try to avoid the conversion somehow (that I was not aware of). This would then not require a bug fix for MSVC.